playwright-tests/tests/payments/create-payment-request.spec.js

```diff
@@ -0,0 +1,113 @@
+import { test, expect } from "@playwright/test";
+import {
+  getTransactionModalObject,
+  mockTransactionSubmitRPCResponses,
+} from "../../util/transaction";
+import { mockRpcRequest } from "../../util/rpcmock";
+import { setDontAskAgainCacheValues } from "../../util/cache";
+
+test.describe("admin connected", function () {
+  test.use({
+    storageState: "playwright-tests/storage-states/wallet-connected-admin.json",
+  });
+  test("create payment request", async ({ page }) => {
+    await page.goto("/treasury-devdao.near/widget/app?page=payments");
```

Comment:
should we also add test for manual request?

Reply:
yes, I will add that too

Reply:
@Megha-Dev-19 See the test ( you can find the video in the description ) for creating manual payment request

Reply:
@petersalomonsen it looks great